src/_transformations/vendor.ts

```diff
@@ -1,20 +1,19 @@
 import { Node, SourceFile } from "../deps.deno.ts";
 import type { Context } from "../context.ts";
-import * as cache from "../cache.ts";
 
 interface LocalFile {
   readonly sourceFile: SourceFile;
   readonly url: URL;
 }
 
 function getVendoredPath(url: URL) {
+  // FIXME respect ${vendorDir}/import_map.json
```

Comment:
I guess this should be fixed before merging?

Otherwise, I very much approve of this!

Reply:
Import map support is non-trivial. It deserves a separate PR.

Reply:
To correctly resolve relative imports within vendored modules, we need to map them back to remote URLs; I'm not sure how to do that yet